complete-cats-config.json

```diff
@@ -0,0 +1,92 @@
+IN DEVELOPMENT
```

Comment:
I think we can remove "IN DEVELOPMENT" for both files.

Reply:
done